src/drivers/distance_sensor/vl53l1x/vl53l1x.cpp

```diff
@@ -222,35 +232,55 @@ void VL53L1X::RunImpl()
 {
 	uint8_t dataReady = 0;
 
+	uint8_t roiCenter[] = {247, 215, 183, 167, 151}; 
```

Comment:
Comment explain these magic numbers?

Is it desirable to configure a specific ROI for different usage?

Reply:
logic to these numbers can be found in vl53l1x multiple detection zone pdf.

Reply:
Please use defines to encode them.